indico/modules/events/papers/controllers/management.py

```diff
@@ -226,3 +228,34 @@ def _process(self):
                                num_not_submitted_papers).format(num=num_not_submitted_papers), 'warning')
             return jsonify_data(**self.list_generator.render_list())
         return jsonify_form(form=form, submit=_('Judge'), disabled_until_change=False)
+
+
+class RHManageReviewingSettings(RHManagePapersBase):
+    def _process(self):
+        defaults = FormDefaults(content_review_questions=self.event_new.cfp.content_review_questions,
+                                layout_review_questions=self.event_new.cfp.layout_review_questions,
+                                **paper_reviewing_settings.get_all(self.event_new))
+        form = PaperReviewingSettingsForm(event=self.event_new, obj=defaults)
+        if form.validate_on_submit():
+            data = form.data
+            content_review_questions = data.pop('content_review_questions', [])
```

Comment:
The whole `*_review_questions` logic here is still not needed. See `RHManageAbstractReviewing`